prototype/prototype_test.go

```diff
@@ -147,3 +149,45 @@ func TestSearch(t *testing.T) {
 	})
 	assertSearch(t, idx, Substring, "foo", []string{})
 }
+
+func TestApiVersionValidate(t *testing.T) {
```

Comment:
Is this the same test that was repeated above? (I think twice) Would it be possible to consolidate this to one place place?

Reply:
They could, the reason I didn't is: (1) these versions will proceed independently, so the ROI for sharing the test code seems small, and (2) share the test code seems awkward -- in which package should it go? We could re-arrange the way the packages are organized, but I am not sure that this is worth the ROI either.

DO you have a specific idea in mind you'd like to try?